graphql/rules.lua

```diff
@@ -31,13 +31,16 @@ end
 function rules.fieldsDefinedOnType(node, context)
   if context.objects[#context.objects] == false then
     local parent = context.objects[#context.objects - 1]
+    if(parent.__type == 'List') then
+      parent = parent.ofType
+    end
     error('Field "' .. node.name.value .. '" is not defined on type "' .. parent.name .. '"')
```

Comment:
I see where you are going but i think the message is correct now since "field" is part of the "parent" definition and has nothing to do with the list. For example:
```"is_complete" is a property of "task" type``` sounds good but
```"is_complete" is a property of "[task]" type``` is a bit wired